src/connectionbenchmark/benchmarktaskdownload.h

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BENCHMARKTASKDOWNLOAD_H
+#define BENCHMARKTASKDOWNLOAD_H
+
+#include "benchmarktask.h"
+#include "networkrequest.h"
```

Comment:
remove this. Use a forward declaration: `class NetworkRequest;` just before `class BenchmarkTaskDownload...`